ansible/roles/ocp4-workload-pipelines/tasks/remove_workload.yml

```diff
@@ -0,0 +1,69 @@
+---
+# Implement your Workload removal tasks here
+
+- name: Set up ocp4_workload_pipelines combined dictionary
+  set_fact:
+    ocp4_workload_pipelines: >-
+      {{ ocp4_workload_pipelines_defaults
+       | combine(ocp4_workload_pipelines_vars    | default( {} ),
+                 ocp4_workload_pipelines_secrets | default( {}), recursive=true )
+      }}
+
+- name: Remove Pipelines Installation
+  k8s:
+    state: absent
+    api_version: operator.tekton.dev/v1alpha1
+    kind: Config
+    name: cluster
+
+- name: Get Installed CSV
+  k8s_facts:
+    api_version: operators.coreos.com/v1alpha1
+    kind: Subscription
+    name: openshift-pipelines-operator
+    namespace: openshift-operators
+  register: r_subscription
+
+- name: Print CSV
+  debug:
+    msg: "CSV: {{ r_subscription }}"
+- name: Remove CSV
```

Comment:
formatting nit: add line above

Also, do you need to remove the CSV or will removing the subscription take care of this also?

Reply:
Need to. Removing just the subscription leaves the CSV.